api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -673,7 +674,14 @@ ml_single_get_tensors_info (ml_single_h single, gboolean is_input,
 
   ML_SINGLE_GET_VALID_HANDLE_LOCKED (single_h, single, 0);
 
-  ml_single_get_tensors_info_from_filter (single_h->filter, is_input, info);
+  /* allocate handle for tensors info */
+  ml_tensors_info_create (info);
```

Comment:
Is `ml_tensors_info_create` always successful?
Otherwise, `*info` may generate a segmentation fault.

Reply:
you are right, updated.